hotshot-query-service/src/data_source/storage/sql.rs

```diff
@@ -790,6 +785,9 @@ impl PruneStorage for SqlStorage {
                             message: format!("failed to commit {e}"),
                         })?;
 
+                        // Note: We don't vacuum the Postgres database,
+                        // as there is no manual trigger for incremental vacuum,
+                        // and a full vacuum can take a lot of time.
```

Comment:
Nit: this comment might be more appropriate on the implementation of `fn vacuum`, as it's really explaining why the `#[cfg(feature = "embedded-db")]` is there